index.html

```diff
@@ -0,0 +1,293 @@
+    <script src="https://kit.fontawesome.com/df4dde8176.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.7.2/font/bootstrap-icons.css">
+    <link rel="stylesheet" href="./css/style.css">
+    <title>FOOD INDUSTRY 29-31.03.2022</title>
+</head>
+<body>
+    <div class="prefer-menu hidden">
+        <i class="far fa-times-circle fa-3x exit-btn"></i>
+        <ul class="menu-prefer">
+            <li><a class="link menu-open-home" href="#home">HOME</a></li>
+            <li><a class="link" href="#program">Program</a></li>
+            <li><a class="link menu-open-about" href="#about">About</a></li>
+            <li><a class="link partner-link" href="#sponsor">Sponsor1</a></li>    
+        </ul>
+    </div>
+    
+    <main class="main" id="home">
+    
+        <nav class="nav">
+            <ul class="nav-top-list">
+                <li><a href="https://www.facebook.com/oybek.kayumov.39/" target="_blank" rel="noopener noreferrer"><i class="fab fa-facebook fa-lg"></i>
+                    </a></li>
+                <li><a href="https://twitter.com/KayumovOybek" target="_blank" rel="noopener noreferrer">
+                        <i class="fab fa-twitter fa-lg"></i>
+                    </a></li>
+                <li><a href="#">English</a></li>
+                <li><a href="#">My Page</a></li>
+                <li><a href="#">Logout</a></li>
+            </ul>
+            <div class="nav-bottom">
+                <a class="logo" href="#home"> <img alt="logo" src="images/logo.png" height="90" width="180">
+                </a>
+                <ul class="nav-bottom-list">
+                    <li><a class="link about-link" href="#about">About</a></li>
+                    <li><a class="link" href="#program">Program</a></li>
+                    <li><a class="link" href="#join">Join</a></li>
+                    <li><a class="link" href="#partner">Sponsor</a></li>
+                    <li><a class="link" href="#news">News</a></li>
+                    <li class="nav-list-button"><a class="link" href="#home">CC Campaign</a></li>
+                </ul>
+            </div>
+        </nav>
+    
+        <header class="head">
+            <i class="fas fa-bars fa-3x hamburger open"></i>
+            <h1 class="head-top">"21st International Exhibition"</h1>
+            <h2 class="head-main">Food, Ingredients and Food Technologies</h2>
+            <p class="head-p">
+                UzFood is the key event in the food industry. Every year the exhibition is attended by leading producers, suppliers and distributors of the food industry representing both national and global food markets. <br> The exhibition is annually visited by top managers of the leading companies, importers and distributors.
+            </p>
+            <h2 class="head-data">2022.03.29(TUE) ~ 31(THU)</h2>
+            <h2 class="head-place">@Uzexpocentre, Tashkent, Uzbekistan</h2>    
+        </header>
```

Comment:
- [ ] To match the original design better, you should consider double-checking your `background-image`. Right now there is a white notch on the top of your header.
- [ ] Beware that the same thing happens on the about page!

Original design:

![image](https://user-images.githubusercontent.com/66967274/147248445-7a2e0731-26bd-4ac6-9d03-dac6dcccbea0.png)


Your design:
![image](https://user-images.githubusercontent.com/66967274/147248380-66232044-707f-46a9-b128-a0046c798038.png)

Reply:
updated